lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -700,10 +700,9 @@ static bool hasExplicitSendableConformance(NominalTypeDecl *nominal,
 }
 
 /// Find the import that makes the given nominal declaration available.
-static Optional<AttributedImport<ImportedModule>> findImportFor(
-    NominalTypeDecl *nominal, const DeclContext *fromDC) {
+Optional<AttributedImport<ImportedModule>> findImportFor(
+    ModuleDecl *nominalModule, ASTContext *nominalASTCtx, const DeclContext *fromDC) {
```

Comment:
You shouldn't need to pass the ASTContext as a separate parameter, you can get it from the module e.g `nominalModule->getASTContext()`

Reply:
Wouldn’t the ASTContext of the parentModule & the ASTContext of the nominalTypeDecl be different?

Reply:
`ASTContext` is effectively global state for a compilation, so it'll be the same

Reply:
Ooh I see, great! I've updated it